src/components/ImageView/Overlay/OverlayComponent.tsx

```diff
@@ -105,15 +104,15 @@ export class OverlayComponent extends React.Component<OverlayComponentProps> {
                 currentStyleString += `, Tol=${tolVal}`;
             }
 
-            if (!this.props.frame.validWcs) {
+            if (!frame.validWcs) {
                 //Remove system and format entries
                 currentStyleString = currentStyleString.replace(/System=.*?,/, "").replaceAll(/Format\(\d\)=.*?,/g, "");
             }
 
             if (!settings.title.customText) {
-                currentStyleString += `, Title=${frame.filename}`;
-            } else if (frame.titleCustomText?.length) {
-                currentStyleString += `, Title=${frame.titleCustomText}`;
+                currentStyleString += `, Title=${this.props.image?.store?.filename}`;
```

Comment:
We don't use `frame` here because we need to use `ColorBlendingStore.filename` instead of the base frame's filename.
I think we don't need a if-statement block because we are applying the same behaviour for loaded images and blended images: default titles (the filename) and custom titles (changeable in the settings).